front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/ToolboxItem.tsx

```diff
@@ -20,6 +20,7 @@ const StyledBox = styled(Box)`
 const ToolboxItem = ({ icon, label }: Props) => {
   return (
     <StyledBox
+      height="48px"
```

Comment:
This will be problematic if in some language it needs two rows to display. Configure via paddings/margins instead :)